packages/job-worker/src/workers/events/child.ts

```diff
@@ -31,10 +31,11 @@ export class EventsWorkerChild {
 	constructor(
 		emitLockEvent: (event: AnyLockEvent) => Promise<void>,
 		queueJob: QueueJobFunc,
-		logLine: LogLineFunc,
+		logLine: LogLineWithSourceFunc,
 		fastTrackTimeline: FastTrackTimelineFunc | null
 	) {
-		interceptLogging(logLine)
+		// Intercept logging to pipe back over ipc
+		interceptLogging('events-child', logLine)
```

Comment:
This will lose the context of which 'events-child' thread it was. was it studio0 or studio1?
I assume its the same for the other threads